pages/attached_result.html

```diff
@@ -38,16 +38,13 @@
                         getValues: function() {
                             return attachedIds;
                         }
-                    },
```

Comment:
Why is it modified?

Reply:
I removed the stuff that was there to test user actions

Reply:
I removed the stuff that was there to test user actions